file-formats/rvbc/type/zif_aff_rvbc_v1.intf.abap

```diff
@@ -0,0 +1,122 @@
+      "! <p class="shorttext">Supported Only for Predefined Analytical Queries</p>
+      "! Supported only for predefined analytical queries
+      supported_predefined_queries TYPE ty_extensibility_mode VALUE 1,
+      "! <p class="shorttext">Not Supported</p>
+      "! Not supported
+      not_supported                TYPE ty_extensibility_mode VALUE 2,
+    END OF co_extensibility_mode.
+
+  "! $values { @link zif_aff_rvbc_v1.data:co_source }
+  "! $default { @link zif_aff_rvbc_v1.data:co_source.predefined }
+  TYPES ty_source TYPE n LENGTH 1.
+  CONSTANTS:
+    BEGIN OF co_source,
+      "! <p class="shorttext">Predefined</p>
+      "! Predefined
+      predefined TYPE ty_source VALUE 1,
+      "! <p class="shorttext">Custom</p>
+      "! Custom
+      custom     TYPE ty_source VALUE 2,
+    END OF co_source.
+
+  "! $values { @link zif_aff_rvbc_v1.data:co_status }
+  "! $default { @link zif_aff_rvbc_v1.data:co_status.draft }
+  TYPES ty_status TYPE n LENGTH 1.
+  CONSTANTS:
+    BEGIN OF co_status,
+      "! <p class="shorttext">Draft</p>
+      "! Draft
+      draft TYPE ty_status VALUE 1,
+      "! <p class="shorttext">Final</p>
+      "! Final
+      final TYPE ty_status VALUE 2,
+    END OF co_status.
+
+  "! $values { @link zif_aff_rvbc_v1.data:co_consistency_status }
+  "! $default { @link zif_aff_rvbc_v1.data:co_consistency_status.check_required }
+  TYPES ty_consistency_status TYPE n LENGTH 1.
+  CONSTANTS:
+    BEGIN OF co_consistency_status,
+      "! <p class="shorttext">Consistency Check Is Required</p>
+      "! Consistency check is required
+      check_required TYPE ty_consistency_status VALUE 0,
+      "! <p class="shorttext">Consistent</p>
+      "! Consistent
+      consistent     TYPE ty_consistency_status VALUE 1,
+      "! <p class="shorttext">Error</p>
+      "! Error
+      error          TYPE ty_consistency_status VALUE 2,
+    END OF co_consistency_status.
+
+  TYPES:
+    "! <p class="shorttext">Review Booklet Definition</p>
+    "! Review booklet definition
+    BEGIN OF ty_booklet_definition,
+      "! <p class="shorttext">Application ID</p>
+      "! Application ID
+      "! $required
+      application        TYPE ty_app_id,
+      "! <p class="shorttext">Source</p>
+      "! Source
+      source             TYPE ty_source,
+      "! <p class="shorttext">Status</p>
+      "! Status
+      status             TYPE ty_status,
+      "! <p class="shorttext">Consistency Status</p>
+      "! Consistency status
+      consistency_status TYPE ty_consistency_status,
+      "! <p class="shorttext">Extensibility Mode</p>
+      "! Extensibility mode
+      "! $required
+      extensibility_mode TYPE ty_extensibility_mode,
+    END OF ty_booklet_definition.
+
+  TYPES:
+    "! <p class="shorttext">InA Service</p>
+    "! InA Service
+    BEGIN OF ty_ina1_service,
+      "! <p class="shorttext">Service Name</p>
+      "! Service name
+      "! $required
+      ina1_service TYPE ty_ina1_service_name,
```

Comment:
Usually, we try to keep the name of the tile and the field name identical. So either "Service Name" or "INA1 Service".